source/stylesheets/main.css.scss

```diff
@@ -19,6 +38,10 @@ $mq-breakpoint-widescreen: 1200px;
   }
 }
 
+.app-c-site-width-container--constraint {
```

Comment:
It is used in the error layouts. I thought it's part of the layouts that's why I added it to this commit, otherwise happy to move it to https://github.com/alphagov/govuk-design-system/pull/77